src/UnderlineNav2/UnderlineNav.Item.stories.tsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react'
+import {Meta, Story} from '@storybook/react'
+import {BaseStyles, ThemeProvider} from '..'
+import {UnderlineNav} from './index'
+import {UnderlineNavItem} from './UnderlineNavItem'
+import {CodeIcon, GitPullRequestIcon, PeopleIcon} from '@primer/octicons-react'
+
+const icons = {CodeIcon, GitPullRequestIcon, PeopleIcon}
+
+export default {
+  title: 'Drafts/Components/UnderlineNav/UnderlineNav.Item',
+  component: UnderlineNavItem,
+  decorators: [
+    Story => {
+      return (
+        <ThemeProvider>
+          <BaseStyles>
+            <UnderlineNav aria-label="Repository">
+              <Story />
+            </UnderlineNav>
+          </BaseStyles>
+        </ThemeProvider>
+      )
+    }
+  ],
+  parameters: {
+    controls: {
+      expanded: true,
+      exclude: ['as']
+    }
+  },
+  args: {
+    children: 'Code',
+    counter: '12K',
+    icon: PeopleIcon
+  },
+  argTypes: {
+    children: {
+      type: 'string'
+    },
+    counter: {
+      type: 'string'
+    },
+    icon: {
+      options: Object.keys(icons),
```

Comment:
It would be so cool if we would write some kind of helper for this so all stories have access to a dropdown for octicons 😄 for now we'll have to keep copy/pasting this object

Reply:
I'll look into that as well ✨

Reply:
I wrote something for this. See this [commit](https://github.com/primer/react/pull/2485/commits/e711ca42f017baaa32c2137bd99a44049aec842a)

Still we need to pass what icons we want to show into the helper function, I thought if we display all icons, it might be too many  😄  let me know what you think!

Reply:
😮 I love it! This is fantastic ✨ I agree I think its nice to pass in just a few icons! 🚀